Validator/Validator/OnTheFlyFormatConverterValidator.php

```diff
@@ -11,6 +11,9 @@
 
 namespace Integrated\Bundle\ImageBundle\Validator\Validator;
 
+use Doctrine\Common\Collections\ArrayCollection;
+use Integrated\Bundle\SolrBundle\Process\Exception\LogicException;
```

Comment:
What is the reason that the LogicException from the solr bundle, which is not even in the composer.json, is used ?